Library/Homebrew/vendor/README.md

```diff
@@ -35,7 +35,7 @@ Vendored Dependencies
 ### ruby-macho
 
 > The MIT License
-> Copyright (c) 2015, 2016, 2017 William Woodruff <william @ tuffbizz.com>
+> Copyright (c) 2015, 2016, 2017, 2018 William Woodruff <william @ yossarian.net>
```

Comment:
Can skip the dates here FYI; they have no legal bearing.

Reply:
Alternatively, it just needs to include the first date. Copyright is considered active *from* the inception date, it does not need to be updated to cover every year.

Reply:
I updated the dates/email in a commit to ruby-macho (https://github.com/Homebrew/ruby-macho/commit/7a9f1f26f3e5cbf6720646a0caf59d5470a18f29#diff-9879d6db96fd29134fc802214163b95a), so I figured it made sense to synchronize the LICENSE text there with the text here. Going forwards, though, I'll stop adding new dates.